File: Traversal/Java/Even_Odd_Sum.java
class Solution {
    ArrayList<Integer> EvenOddSum(int N, int Arr[]) {
        // code here
        int even=0, odd=0;
        for (int i=0; i<N; i++) {
            if (i%2 == 0) {
                odd += Arr[i];
            }
            else {
                even += Arr[i];
            }
        }
        ArrayList<Integer> a=new ArrayList<>();
        a.add(odd);
        a.add(even);
        return a;
    }
}

File: Solutions/Python/Even_Odd_Sum.py
class Solution:
    def EvenOddSum(self,N,Arr):
        #code here (return list)
        oddSum = 0
        evenSum = 0
        for idx in range(N):
            if idx%2 != 0:
                oddSum += Arr[idx]
            else:
                evenSum += Arr[idx]
        return evenSum, oddSum
